integrationtesting/fixtures/src/main/resources/META-INF/services/org.opendcs.fixtures.spi.ConfigurationProvider

```diff
@@ -1,4 +1,4 @@
 org.opendcs.fixtures.configurations.xml.XmlConfigurationProvider
 org.opendcs.fixtures.configurations.opendcs.pg.OpenDCSPGConfigurationProvider
-org.opendcs.fixtures.configurations.opendcs.oracle.OpenDCSOracleConfigurationProvider
```

Comment:
NOTE: There *is* and OpenDCSOracleConfigurationProvider. Not 2 CwmsOracleConfigurationProviders.

Reply:
bad merge conflict resolution - added back in